cloud_info/providers/openstack.py

```diff
@@ -341,7 +354,8 @@ def _get_dn(self, x509name):
 
         return obj_name[start:end]
 
-    def get_compute_endpoints(self, **kwargs):
+    @_rescope
+    def get_compute_endpoints(self, os_project_name=None, **kwargs):
```

Comment:
@alvarolopez this change (and the similar ones on `get_templates` and `get_images` are not coherent with the base class [providers/__init.py__](https://github.com/EGI-Foundation/cloud-info-provider/blob/d07d879094e631c26e0f051eaa9721f8fa7139a2/cloud_info/providers/__init__.py). Is there any clean way to easily handle this?